### PR TITLE
EMQX 11315 feat gw jt808 docs

### PR DESCRIPTION
**Pull request by JimMoen**

### CODE REVIEW

**Comment by Meggielqk** [comment]
@wivwiv ，之前的 configuration manual 的地址在目录文件里已经被替换为新的配置页面了，导致原先文档里的"../configuration/configuration-manual,md"链接失效。如果替换为“[网关配置 - JT/T 808 网关](https://docs.emqx.com/zh/enterprise/v${EE_VERSION}/hocon/)”，合并之后会生效吗？

File: zh_CN/gateway/jt808.md [hunk 1]
```diff
@@ -1,31 +1,44 @@
-# JT/T 808 协议网关
+# JT/T808 协议网关
```
Comment (JimMoen) [inline zh_CN/gateway/jt808.md]: @Meggielqk keep space between `JT/T` and `808`.
That's the standard protocol name.
> `JT/T 808`

Comment (Meggielqk) [reply zh_CN/gateway/jt808.md]: OK. When we refer to the gateway in Dashboard, shall we use "JT/T808 gateway" to be consistent with the Dashboard name?

Comment (JimMoen) [reply zh_CN/gateway/jt808.md]: `JT/T 808` should also be used in Dashboard.
https://github.com/emqx/emqx-dashboard5/pull/2224

Comment (Meggielqk) [reply zh_CN/gateway/jt808.md]: I have asked Shifan to change to "JT/T 808".

**Review by wivwiv — DISMISSED** [review]

**Review by wivwiv — APPROVED** [review]

**Pull request merged**